client/app/bundles/course/assessment/pages/AssessmentStatistics/AnswerDetails/AnswerDetails.tsx

```diff
@@ -0,0 +1,81 @@
+import { defineMessages } from 'react-intl';
+import { Card, CardContent } from '@mui/material';
+import { QuestionType } from 'types/course/assessment/question';
+import { AnswerDetailsMap } from 'types/course/statistics/answer';
+import { QuestionDetails } from 'types/course/statistics/assessmentStatistics';
+
+import useTranslation from 'lib/hooks/useTranslation';
+
+import FileUploadDetails from './FileUploadDetails';
+import MultipleChoiceDetails from './MultipleChoiceDetails';
+import MultipleResponseDetails from './MultipleResponseDetails';
+import TextResponseDetails from './TextResponseDetails';
+
+const translations = defineMessages({
+  rendererNotImplemented: {
+    id: 'course.assessment.submission.Answer.rendererNotImplemented',
+    defaultMessage:
+      'The display for this question type has not been implemented yet.',
```

Comment:
What's the difference between
`course.assessment.submission.Submission.Answer.rendererNotImplemented`
and
`course.assessment.submission.Submission.SubmissionAnswer.rendererNotImplemented

Reply:
actually the inconsistencies already occur quite long ago, starting from when SubmissionAnswer was being refactored to Answer (that time, the focus was to make the implementation of Answer to be more concise and more expandable towards the future attributes, and hence the translation part is overlooked).

Now after the thorough observation, it's only natural to change the translation all from SubmissionAnswer to Answer, to comply with the new naming of that component, which is Answer